pandas/io/parsers/base_parser.py

```diff
@@ -1144,14 +1144,20 @@ def converter(*date_cols, col: Hashable):
                 date_format.get(col) if isinstance(date_format, dict) else date_format
             )
 
-            result = tools.to_datetime(
-                ensure_object(strs),
-                format=date_fmt,
-                utc=False,
-                dayfirst=dayfirst,
-                errors="ignore",
-                cache=cache_dates,
-            )
+            with warnings.catch_warnings():
+                warnings.filterwarnings(
+                    "ignore",
+                    ".*parsing datetimes with mixed time zones will raise a warning",
+                    category=FutureWarning,
+                )
+                result = tools.to_datetime(
```

Comment:
there's a couple more usages of `to_datetime` in this function, I think we should ideally silence them too

Reply:
Thank you, I added` catch_warnings() `in two other places where we call `to_datetime` in this function.
I get failures in ci and I’m not sure if they are related to my changes. Locally all tests pass.

e.g the tests below failed in ci, but passed locally:

```
pandas/tests/arrays/floating/test_construction.py
pandas/tests/arrays/integer/test_repr.py
pandas/tests/arrays/floating/test_repr.py
```

we don't even call this function (`converter()`) while running these tests.